packages/react-router/lib/components.tsx

```diff
@@ -51,6 +51,7 @@ import {
 } from "./hooks";
 
 export interface FutureConfig {
+  v7_partialHydration: boolean;
```

Comment:
This can be left off - it's nuanced and annoying but there are currently 2 sets of future flags:
* Those that apply to the router via `createRouter`
* Those that apply to the react layer via `RouterProvider`

This is the latter and partialHydration is the former

Reply:
We also discussed during the burst that the `v7_` prefix is superfluous, but if we already have a couple future flags that use `v7_` prefix then we can keep it for now and just remember not to include `v8_` prefixes in next version